packages/core/src/RenderingEngine/helpers/createVolumeActor.ts

```diff
@@ -48,45 +49,40 @@ async function createVolumeActor(
   const volumeActor = vtkVolume.newInstance();
   volumeActor.setMapper(volumeMapper);
 
-  const voiRange = volumeActor
-    .getProperty()
-    .getRGBTransferFunction(0)
-    .getRange()
-    .slice();
+  // If the volume is composed of imageIds, we can apply a default VOI based
+  // on either the metadata or the min/max of the middle slice.
+  if (imageVolume.imageIds) {
+    await setDefaultVolumeVOI(volumeActor, imageVolume);
```

Comment:
Hmm, i don't think we should await here, this is blocking the creation of the volumeActor on the frame request to set the VOI.

We should instead pass through and return the volumeActor, and only the callback + triggerVOIModified should await the frame request?